sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2190,31 +2206,32 @@ def _execute(
             # Case: task_lib.Task(run, num_nodes=1)
             self._execute_task_one_node(handle, task, job_id, detach_run)
 
-    def _post_execute(self, handle: ResourceHandle, teardown: bool) -> None:
+    def _post_execute(self, handle: ResourceHandle) -> None:
```

Comment:
I think we should keep the `teardown` arg, for reason below:

1. I ran `sky launch --down -c myclus minimal.yaml` and alt-tabbed to something else
2. Came back to my terminal, saw messages:
```
I 10-23 19:28:56 cloud_vm_ray_backend.py:2111] Job ID: 1
I 10-23 19:28:56 cloud_vm_ray_backend.py:2111] To cancel the job:       sky cancel myclus 1
I 10-23 19:28:56 cloud_vm_ray_backend.py:2111] To stream the logs:      sky logs myclus 1
I 10-23 19:28:56 cloud_vm_ray_backend.py:2111] To view the job queue:   sky queue myclus
I 10-23 19:28:56 cloud_vm_ray_backend.py:2221]
I 10-23 19:28:56 cloud_vm_ray_backend.py:2221] Cluster name: myclus
I 10-23 19:28:56 cloud_vm_ray_backend.py:2221] To log into the head VM: ssh myclus
I 10-23 19:28:56 cloud_vm_ray_backend.py:2221] To submit a job:         sky exec myclus yaml_file
I 10-23 19:28:56 cloud_vm_ray_backend.py:2221] To stop the cluster:     sky stop myclus
I 10-23 19:28:56 cloud_vm_ray_backend.py:2221] To teardown the cluster: sky down myclus
```
3. Tried the suggested commands, they didn't work:
```
(base) romilb@romilbx1yoga:/mnt/d/Romil/Berkeley/Research/sky-experiments/examples$ ssh myclus
ssh: connect to host 18.232.157.190 port 22: Connection refused
(base) romilb@romilbx1yoga:/mnt/d/Romil/Berkeley/Research/sky-experiments/examples$ sky cancel myclus 1
sky.exceptions.ClusterNotUpError: Cluster 'myclus' does not exist. skipped.
```

Since we set an auto-down of 1 minute, for `--down`, which is too short for the above commands to be useful, maybe we should not print them.